src/main/java/org/broadinstitute/hellbender/tools/exome/CnvCaller.java

```diff
@@ -52,39 +55,39 @@
-     *
-     * @param targets the collection representing all targets
-     * @param segments segments, each of which holds a reference to these same targets
+     * Make calls for a list of segments based on the coverage data in a set of targets and their tangent-normalized coverage.
+     *  @param normalizedCoverage the collection representing targets and their tangent-normalized coverage
+     * @param segments segments, each of which holds a reference to these same normalizedCoverage
      * @param zThreshold number of standard deviations from mean required to call an amplification or deletion
      */
-    public static List<ModeledSegment> makeCalls(final TargetCollection<TargetCoverage> targets, final List<ModeledSegment> segments, final double zThreshold) {
+    public static List<ModeledSegment> makeCalls(final ReadCountCollection normalizedCoverage, final List<ModeledSegment> segments, final double zThreshold) {
         Utils.nonNull(segments, "Can't make calls on a null list of segments.");
-        Utils.nonNull(targets, "Can't make calls on a null list of targets.");
+        Utils.nonNull(normalizedCoverage, "Can't make calls on a null list of normalizedCoverage.");
         ParamUtils.isPositiveOrZero(zThreshold, "zThreshold must be positive or zero.");
 
         logger.warn("This caller is still considered experimental.");
 
         /**
-         * estimate the copy-number neutral log_2 coverage as the median over all targets.
+         * estimate the copy-number neutral log_2 coverage as the median over all normalizedCoverage.
          * As a precaution we take the median after filtering extreme coverages that are definitely not neutral
          * i.e. those below -NON_NEUTRAL_COVERAGE or above +NON_NEUTRAL_COVERAGE
+         *
+         * Note that we use column 0, implicitly assuming that there is only one sample / column
          */
-        final double []  nonExtremeCoverage = targets.targets()
-                .stream()
-                .mapToDouble(TargetCoverage::getCoverage)
-                .filter(c -> Math.abs(c) < NON_NEUTRAL_THRESHOLD)
-                .toArray();
+        final double []  nonExtremeCoverage = Arrays.stream(normalizedCoverage.counts().getColumn(0))
+                .filter(c -> Math.abs(c) < NON_NEUTRAL_THRESHOLD).toArray();
         final double neutralCoverage = new Median().evaluate(nonExtremeCoverage);
 
         logger.info(String.format("True copy neutral estimate (median): %.3f", neutralCoverage));
         logger.info(String.format("True copy neutral estimate (median) in CR: %.3f", Math.pow(2, neutralCoverage)));
 
-        // Get the standard deviation of targets belonging to high-confidence neutral segments
+        // Get the standard deviation of normalizedCoverage belonging to high-confidence neutral segments
+        final TargetCollection<ReadCountRecord.SingletonRecord> targetsWithCoverage =
```

Comment:
See above -- basically, `TargetCollection` has a method to get all targets overlapping a given `Locatable`, which `ReadCountCollection` lacks.  It probably makes sense to refactor `ReadCountCollection` in this way, but I would prefer to handle that separately.  In fact, I would enjoy doing this, but there's something to be said for smaller PRs.